src/styles/util.scss

```diff
@@ -1,24 +1,71 @@
 @use "./mixins/a11y";
 @use "./mixins/layout";
+@use "./mixins/utils";
+@use './variables' as var;
 
-// Utility classes for layout
-.u-stretch {
-  flex-grow: 1;
-}
+// Utility classes
+// Use these classes on elements that don't require additional styling
 
+// Layout
+
+// Establish a flex container with a horizontal flow
 .u-layout-row {
   @include layout.row;
 }
 
+// Establish a row flex container that aligns its children to (vertical) center
 .u-layout-row--align-center,
 .u-layout-row--align-middle {
   @include layout.row($align: center);
 }
 
+// Establish a row flex container that aligns its children along the baseline
+.u-layout-row--align-baseline {
+  @include layout.row($align: baseline);
+}
+
+// Establish a row flex container justifies its children right
+.u-layout-row--justify-right {
+  @include layout.row($justify: right);
+}
+
+// Use this class to make an element in a flex container "stretchy" and grow
+// when there is extra space available.
+.u-stretch {
+  flex-grow: 1;
+}
+
+// Put equal, standard spacing between children horizontally
+.u-horizontal-rhythm {
```

Comment:
can you add a comment that the spacing is added to the left of each element.

Reply:
Can you elaborate on why you think this is important to comment? Yes, technically the spacing is applied to the left of the element, but is that important to know? The net effect is that there is space between elements. It does not add spacing to the first element, so the net spacing is even. I'm not pushing back so much as trying to understand the need here.